specification/_global/search/_types/highlighting.ts

```diff
@@ -193,5 +193,5 @@ export enum HighlighterType {
 export class HighlightField extends HighlightBase {
   fragment_offset?: integer
   matched_fields?: Fields
-  analyzer?: Analyzer
+  analyzer?: string
```

Comment:
None of the examples in https://www.elastic.co/guide/en/elasticsearch/reference/current/highlighting.html suggest that it could be anything else than a string.

Reply:
Thanks for this! I had totally missed that the examples in the highlighting docs were actually index mappings.

Reply:
Done in [`5b6d7ea` (#2796)](https://github.com/elastic/elasticsearch-specification/pull/2796/commits/5b6d7ea8852efd1f2f0f6e19eff441a3f1d140a4). Please take another look.